docs/operator-guides/configuring-airbyte-db.md

```diff
@@ -0,0 +1,81 @@
+# Configuring Airbyte Internal Database
+
+Airbyte uses different objects to store internal states and metadata. This data is stored and manipulated by the various Airbyte components, but you have the ability to manage how to deploy this database by using either:
+- By default, Airbyte spins-up its own Postgres database as part of one of th Docker service as described in its `docker-compose.yml` file: `airbyte/db`.
+- Dedicated custom Postgres instance.
+
+The various entities persisted in this internal database can be categorized as:
+
+1. Jobs Database: Data about executions of Airbyte Jobs and various runtime metadata.
+2. Temporal Database: Data about the internal orchestrator used by Airbyte, Temporal.io (Tasks, Workflow data, Events, and visibility data).
+3. Connection Configuration Database: Connectors, Sync Connections and various Airbyte configuration objects.
+
+Note that no actual data from the source (or destination) connectors ever transits or is retained in this internal database.
+
+If you need to interact with it, for example to make back-ups or perform some clean-up maintenances, you can also access to Export and Import functionalities of this database via API, or the UI (in the Admin page, in the Configuration Tab).
+
+## Connecting to some Postgres database
```

Comment:
```suggestion
## Connecting to an External Postgres database
```

Reply:
Is it worth mentioning in this section that if someone is using an external database then they can remove the `db` service from their docker-compose?